resources/assets/coffee/react/profile-page/header-extra.coffee

```diff
@@ -21,8 +21,8 @@ el = React.createElement
 
 bn = 'profile-header-extra'
 
-rowValue = (value) ->
-  "<strong>#{value}</strong>"
+rowValue = (value, className = '', title = '') ->
+  "<strong class='#{className}' title='#{title}'>#{value}</strong>"
```

Comment:
this is gonna result in `class='' title=''` when `class` and `title` aren't given